### PR TITLE
Fixed code compliance

### PR DESCRIPTION
**Pull request by Tom-Kingstone**
### Issues addressed by this PR

Closes #

made methods extension methods

### CODE REVIEW

**Comment by Tom-Kingstone** [comment]
@BHoMBot check required

File: HTTP_Engine/Convert/ToUrlString.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public static string ToUrlString(string baseUrl, Dictionary<string, object> para
 
         /***************************************************/
 
-        public static string ToUrlString(Dictionary<string, object> data)
+        public static string ToUrlString(this Dictionary<string, object> data)
```
Comment (FraserGreenroyd) [inline HTTP_Engine/Convert/ToUrlString.cs:71]: This shouldn't need the `this` keyword - if compliance is complaining, we need to update Test_Toolkit instead on this occasion

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
The `CustomObject` one is correct though 😄

File: HTTP_Engine/Convert/ToUrlString.cs (hunk 1 above)
Comment (Tom-Kingstone) [reply HTTP_Engine/Convert/ToUrlString.cs:71]: Compliance was complaining for this, so should update test toolkit, I'll revert this change and we can dispensate if it's still complaining as long as there aren't any other issues

**Comment by Tom-Kingstone** [comment]
@BHoMBot check core
@BHoMBot check compliance

**Comment by Tom-Kingstone** [comment]
@BHoMBot check required

**Review by FraserGreenroyd — APPROVED** [review]
As discussed with @Tom-Kingstone this is now fine, and the issue the bot is reporting will be fixed by https://github.com/BHoM/Test_Toolkit/pull/470

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am authorising dispensation to be granted on check ref. 23107109550

**Pull request merged**